x/group/server/query_server.go

```diff
@@ -176,6 +176,10 @@ func (s serverImpl) getProposalsByGroupAccount(ctx types.Context, account sdk.Ac
 	return s.proposalByGroupAccountIndex.GetPaginated(ctx, account.Bytes(), pageRequest)
 }
 
+func (s serverImpl) getAllProposals(ctx types.Context, pageRequest *query.PageRequest) (orm.Iterator, error) {
+	return s.proposalTable.PrefixScan(ctx, pageRequest.Offset, pageRequest.Limit)
```

Comment:
`PrefixScan` is wrongly used here:
- `pageRequest.Key` is being ignored;
- if `pageRequest.Offset` > 0, then the `end` parameter of `PrefixScan` (`pageRequest.Limit`) is wrong (theoritically, it should be `pageRequest.Offset + pageRequest.Limit`)

I'm aware that it should work in the particular context of `TallyVotesInvariant()` (with default `pageRequest` values) but then let's not expose this `getAllProposals` function then and just use directly `s.proposalTable.PrefixScan(ctx, nil, nil)` in `TallyVotesInvariant()`